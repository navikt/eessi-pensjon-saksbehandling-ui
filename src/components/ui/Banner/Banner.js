import React, { Component } from 'react'
import { withTranslation } from 'react-i18next'
import { connect } from 'react-redux'
import { bindActionCreators } from 'redux'

import * as uiActions from '../../../actions/ui'
import './Banner.css'

const mapStateToProps = (state) => {
  return {}
}

const mapDispatchToProps = (dispatch) => {
  return { actions: bindActionCreators(Object.assign({}, uiActions), dispatch) }
}

class Banner extends Component {
  toggleHighContrast () {
    const { actions } = this.props
    actions.toggleHighContrast()
  }

  render () {
    const { t, header } = this.props
    return <div className='c-ui-banner'>
      <a className='c-ui-highcontrast-link mt-1' href='#' onClick={this.toggleHighContrast.bind(this)}>{t('highContrast')}</a>
      <h1 className='typo-undertittel m-0 pt-4 pb-4 text-center appTitle'>{header}</h1>
    </div>
  }
}

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(
  withTranslation()(Banner)
)
