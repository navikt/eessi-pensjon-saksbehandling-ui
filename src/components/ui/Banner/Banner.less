@import '../../../constants';

.c-ui-banner {
  align-self: stretch;
  background-color: @pension-orange-alpha;
  border-bottom: 4px solid @pension-orange;
  margin-bottom: 50px;

  .nav-veileder {
    margin-bottom: -50px;
    margin-top: 60px;
  }
}
