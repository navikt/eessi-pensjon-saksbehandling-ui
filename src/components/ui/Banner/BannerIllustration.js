import React, { Component } from 'react'

class BannerIllustration extends Component {

   render () {

     return <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 370 164' className='header__icon' height='133' width='300'><defs><path id='a' d='M.164 42.342A9.83 9.83 0 0 1 0 40.544C0 18.153 16.43 0 36.7 0c20.266 0 36.697 18.153 36.697 40.544 0 .15-.004.297-.01.444l.095-.001s.042 43.158-36.793 43.158C-.143 84.145 0 98.999 0 98.999V42.345l.164-.003z'></path><path id='c' d='M39.427 0H27.593C21.21 0 0 14.835 0 28.34v35.533h79.226V28.341C79.226 14.835 57.645 0 51.265 0H39.427z'></path><path id='e' d='M2.462 47.922a.848.848 0 0 0 .127-.425v-3.12c-.223-8.68 2.768-12.71 5.229-15.772 0 0-1.496 6.669-.456 5.282 5.89-7.83 33.17-2.467 38.829-12.573.711 5.563-6.43 7.241-6.43 7.241 5.693 1.087 10.626-1.223 11.962-3.965.44 2.236-.912 3.72-2.89 5.249 6.997-1.334 7.11-6.188 7.11-6.188.996 6.438 8.17 3.966 8.17 14.505v9.34c0 .48.377.864.844.864h.837a.855.855 0 0 0 .846-.863v-6.938C67.074 16.256 53.474 0 34.754 0 16.888 0 0 16.207.011 37.663.009 37.68 0 37.7 0 37.718v9.779c0 .478.379.863.843.863h.9a.822.822 0 0 0 .634-.31'></path><path id='g' d='M31.577 28.128c-7.411 0-13.42-2.439-13.42-5.448 0-3.008 6.009-5.447 13.42-5.447 7.412 0 13.42 2.439 13.42 5.447 0 3.01-6.008 5.448-13.42 5.448m.717-13.968c-4.682 0-7.865.028-14.01 3.113C4.08 24.403 0 0 0 0c0 40.143 19.44 49.92 32.08 49.92 12.84 0 33.44-7.77 33.44-49.92-.858.67-6.075 22.257-20.635 16.907-5.652-2.076-7.909-2.747-12.59-2.747z'></path><path id='i' d='M99.84 95.82v15.497H65.52V95.82c0-4.433 5.606-9.495 9.36-11.579h15.6c3.788 2.12 9.36 7.197 9.36 11.58z'></path></defs><g fill='none' fill-rule='evenodd'><path fill='#EFEFEF' d='M1 105h102V1H1z'></path><path fill='#838688' d='M13 9.002a4 4 0 1 1-8-.003 4 4 0 0 1 8 .003M100 9.002A3.998 3.998 0 0 1 96 13a4 4 0 1 1 4-3.998'></path><path fill='#E9C264' d='M44.7 76.775c3.48.696-24.379-.49-28.224 0C12.634 77.27 14.41 52 14.41 52h28.224S41.213 76.088 44.7 76.775'></path><path fill='#DAB25B' d='M43.006 73H14.5c.37 2.347.991 3.912 2.015 3.795 3.854-.455 31.771.636 28.284 0-.874-.157-1.434-1.648-1.793-3.795'></path><path fill='#E9C264' d='M86.7 61.778c3.477.69-24.38-.493-28.223 0C54.632 62.272 56.41 37 56.41 37h28.222s-1.416 24.089 2.068 24.778'></path><path fill='#DAB25B' d='M84.827 58H56.6c.367 2.35.98 3.916 1.998 3.797 3.813-.455 31.454.632 28.005 0-.866-.162-1.424-1.647-1.776-3.797'></path><path fill='#E9C264' d='M52.7 37.779c3.483.688-24.378-.495-28.224 0C20.634 38.267 22.41 13 22.41 13h28.223S49.216 37.087 52.7 37.779'></path><path fill='#DAB25B' d='M50.837 34H22.7c.365 2.348.977 3.917 1.99 3.797 3.803-.456 31.358.632 27.913 0-.86-.159-1.416-1.651-1.766-3.797'></path><path fill='#9BBDCD' d='M223.949 128.871H350.08V3.192H223.95z'></path><path stroke='#403A33' stroke-linecap='round' stroke-linejoin='round' stroke-width='4' d='M223.949 128.871H350.08V3.192H223.95z'></path><path fill='#FFF' d='M318.45 49.61c.036-4.425 3.666-7.98 8.106-7.944 4.44.038 8.008 3.653 7.972 8.077-.037 4.424-3.667 7.982-8.107 7.944-4.44-.037-8.008-3.654-7.972-8.078'></path><path fill='#FFF' d='M308.194 52.297c.028-3.258 2.699-5.876 5.969-5.85 3.27.028 5.897 2.691 5.87 5.95-.028 3.257-2.7 5.875-5.968 5.849-3.27-.028-5.9-2.691-5.871-5.95'></path><path fill='#FFF' d='M298.776 43.547c.036-4.397 3.643-7.932 8.056-7.894 4.414.037 7.96 3.63 7.923 8.028-.037 4.396-3.644 7.93-8.057 7.894-4.412-.036-7.959-3.632-7.922-8.028M323.366 40.185a3.197 3.197 0 0 1 3.229-3.163 3.197 3.197 0 0 1 3.175 3.216 3.197 3.197 0 0 1-3.228 3.164 3.197 3.197 0 0 1-3.176-3.217'></path><path fill='#FFF' d='M312.435 37.71c.028-3.226 2.674-5.82 5.912-5.791 3.237.026 5.84 2.662 5.811 5.889-.026 3.227-2.672 5.82-5.91 5.793-3.237-.028-5.84-2.664-5.813-5.89'></path><path fill='#FFF' d='M311.623 44.868c-.033 3.958 3.159 7.191 7.13 7.224 3.97.033 7.217-3.147 7.25-7.105.034-3.956-3.158-7.189-7.13-7.222-3.97-.033-7.217 3.146-7.25 7.103'></path><path fill='#635E59' d='M324.062 107.288L284.73 55.895l-34.758 60.757z'></path><path fill='#74706C' d='M348.42 83.227l-25.56-42.74-33.03 73.887 58.59-9.475z'></path><path fill='#EDEDEC' d='M223.95 106.242c52.841 10.851 56.327-6.365 126.13-6.365v28.995H223.95v-22.63z'></path><path fill='#FFF' d='M283.272 21.283a8.509 8.509 0 0 1-1.889 11.915c-3.824 2.77-9.177 1.928-11.957-1.882-2.782-3.81-1.935-9.145 1.889-11.915 3.824-2.77 9.177-1.928 11.957 1.882'></path><path fill='#FFF' d='M270.674 21.832c3.784 5.186 2.632 12.446-2.571 16.219-5.207 3.77-12.494 2.623-16.278-2.564-3.784-5.186-2.633-12.448 2.57-16.217 5.208-3.773 12.494-2.625 16.279 2.562'></path><path fill='#FFF' d='M273.509 12.772a5.654 5.654 0 0 1-1.255 7.915c-2.541 1.84-6.098 1.282-7.944-1.25a5.652 5.652 0 0 1 1.255-7.915 5.699 5.699 0 0 1 7.944 1.25'></path><path stroke='#403A33' stroke-linecap='round' stroke-linejoin='round' stroke-width='3' d='M223.95 66.032h63.066V3.192h-63.065z'></path><path stroke='#514C47' stroke-linecap='round' stroke-linejoin='round' stroke-width='3' d='M287.016 66.032h63.066V3.192h-63.066zM223.95 128.871h63.066V66.032h-63.065zM287.016 128.871h63.066V66.032h-63.066z'></path><path stroke='#514C47' stroke-linecap='round' stroke-linejoin='round' stroke-width='6' d='M207.105 128.871h159.821'></path><path stroke='#514C47' stroke-linecap='round' stroke-linejoin='round' stroke-width='4' d='M223.949 128.871H350.08V3.192H223.95z'></path><g><g transform='translate(183.473 23.883)'><mask id='b' fill='#fff'><use xlinkHref='#a'></use></mask><g fill='#8B5F26' mask='url(#b)'><path d='M-13.113-1.46H87.55v115.224H-13.113z'></path></g></g><path fill='#E7E5E2' d='M201.073 102.38c1.493 2.388 5.688 44.353 18.28 44.353 12.596 0 18.282-42.746 18.282-44.97-19.283-11.047-36.562.616-36.562.616z'></path><g transform='translate(180.598 99.911)'><mask id='d' fill='#fff'><use xlinkHref='#c'></use></mask><use fill='#D8A25D' xlinkHref='#c'></use><g mask='url(#d)'><path fill='#A13A28' d='M-48.416 105.573h184.704V-39.195H-48.416z'></path></g></g><path fill='#FFF' d='M200.764 102.528c1.53 2.455 8.404 14.673 18.541 22.2 10.068-7.412 18.911-20.547 18.911-22.834-19.752-11.353-37.452.634-37.452.634z'></path><path fill='#E7E5E2' d='M190.162 64.132s-.154.87-.352.727a2.374 2.374 0 0 0-.887-.18c-1.356-.001-2.46 1.138-2.462 2.544l-.013 9.735c-.003 1.407 1.094 2.55 2.453 2.552a2.38 2.38 0 0 0 1.45-.498c3.284 15.676 14.99 26.747 28.95 26.77 13.96.023 25.703-11.01 29.033-26.675.406.31.904.5 1.449.502 1.358.002 2.46-1.137 2.462-2.543l.015-9.736c.001-1.407-1.096-2.549-2.454-2.552-.313 0-.612.067-.887.178-31.482 3.731-42.463-17.672-43.391-17.673 0 0-13.167 12.325-15.366 16.849z'></path><path fill='#635E59' d='M208.947 73.272c-2.13.159-2.725-2.954-2.084-4.987.12-.385.826-2.14 2.074-2.14 1.246 0 1.797.96 1.873 1.124.917 1.982.467 5.828-1.863 6.003M232.714 73.272c2.13.159 2.726-2.954 2.084-4.987-.12-.385-.826-2.14-2.074-2.14a2.095 2.095 0 0 0-1.873 1.124c-.917 1.982-.467 5.828 1.863 6.003'></path><path fill='#D1BFA3' fillRule='nonzero' d='M221.506 76.85c1.252-.19 2.108-.07 2.414.302 1.156 1.41.786 2.9-1.197 4.124-1.043.643-2.479.87-3.239.536a.738.738 0 0 0-.97.372c-.165.368.003.8.375.962 1.256.551 3.187.247 4.611-.631 2.696-1.663 3.313-4.15 1.561-6.284-.74-.902-2.053-1.086-3.779-.823a.73.73 0 0 0-.615.832.735.735 0 0 0 .839.61z'></path><path fill='#D19E9C' fillRule='nonzero' d='M229.894 87.49a6.63 6.63 0 0 1-.463.82 10.367 10.367 0 0 1-1.566 1.89c-1.882 1.793-4.332 2.83-7.45 2.735-3.04-.092-5.478-1.113-7.39-2.76a10.912 10.912 0 0 1-1.745-1.896 6.63 6.63 0 0 1-.518-.82.739.739 0 0 0-.985-.332.727.727 0 0 0-.336.977c.108.218.319.574.64 1.021a12.368 12.368 0 0 0 1.98 2.153c2.157 1.858 4.916 3.013 8.31 3.116 3.534.107 6.359-1.088 8.514-3.142a11.82 11.82 0 0 0 1.787-2.158c.288-.447.475-.803.57-1.02a.727.727 0 0 0-.38-.96.738.738 0 0 0-.968.376z'></path><g><path fill='#E7E5E2' d='M104.84 98.05c1.593 2.547 6.068 47.31 19.5 47.31 13.435 0 19.5-45.595 19.5-47.968-20.569-11.783-39 .658-39 .658z'></path><path fill='#0F5FA9' d='M125.12 127.42l-12.717-31.98c-6.86 0-29.403 15.668-29.403 30.256v38.384h84.24v-38.384c0-14.588-22.548-30.256-29.404-30.256l-12.716 31.98z'></path><path fill='#3C84C5' d='M112.448 93.02c-3.701 1.276-9.163 4.733-9.163 4.733-.277 11.182 10.215 22.463 10.215 22.463l2.13-7.151 8.018 15.002 4.272.133 7.377-13.556 3.006 7.611c10.123-17.382 8.572-24.696 8.572-24.696-4.042-3.43-9.14-5.239-9.14-5.239l-12.09 31.76-13.197-31.06z'></path><path fill='#E7E5E2' d='M156.613 74.02c-3.928 15.972-16.54 27.66-31.494 27.66-14.952 0-27.564-11.688-31.492-27.659a3.54 3.54 0 0 1-1.267.231c-1.72 0-3.12-1.203-3.12-2.692V60.694c0-1.49 1.4-2.694 3.12-2.694.111 0 .221.005.33.015 2.264-18.53 15.92-32.775 32.43-32.775 16.509 0 30.165 14.246 32.43 32.775.109-.01.219-.015.33-.015 1.724 0 3.12 1.204 3.12 2.694V71.56c0 1.49-1.396 2.692-3.12 2.692-.45 0-.88-.082-1.267-.231z'></path><path fill='#635E59' d='M112.687 62.094c-2.257.17-2.889-3.157-2.21-5.33.13-.412.877-2.286 2.2-2.286 1.32 0 1.904 1.024 1.985 1.2.972 2.119.495 6.23-1.975 6.416M137.88 62.094c2.258.17 2.89-3.157 2.21-5.33-.128-.412-.876-2.286-2.199-2.286-1.32 0-1.904 1.024-1.985 1.2-.972 2.119-.497 6.23 1.974 6.416'></path><path stroke='#D1BFA3' stroke-linecap='round' stroke-width='1.56' d='M125.88 65.15c1.35-.208 2.665-.158 3.283.6 1.505 1.849 1.073 3.984-1.462 5.563-1.267.788-3.06 1.11-4.16.624'></path><path stroke='#593A32' stroke-linecap='round' stroke-width='1.58' d='M135.606 77.606s-2.852 6.532-10.786 6.288c-7.934-.243-10.909-6.288-10.909-6.288'></path><g><g transform='translate(90.8 19)'><mask id='f' fill='#fff'><use xlinkHref='#e'></use></mask><use fill='#F6B873' xlinkHref='#e'></use><g fill='#F6B873' mask='url(#f)'><path d='M-18.177-40.56h104.65V82.68h-104.65z'></path></g></g></g><g><g transform='translate(92.204 58)'><mask id='h' fill='#fff'><use xlinkHref='#g'></use></mask><g fill='#F6B873' mask='url(#h)'><path d='M-21.84-39H85.8V84.24H-21.84z'></path></g></g></g></g><g><g transform='translate(90.8 53.32)'><mask id='j' fill='#fff'><use xlinkHref='#i'></use></mask><use fill='#D8A25D' xlinkHref='#i'></use><g fill='#38A161' mask='url(#j)'><path d='M1.357 181h178.6V0H1.357z'></path></g></g><path fill='#FFBD66' d='M163.288 164.637h18.928l2.964-24.82c-.342-.3-2.34-1.706-2.34-1.505l-2.323 11.931-15.342.497-1.055-12.428s-2.392 1.483-3.12 2.257l2.288 24.068z'></path><path fill='#E7E5E2' d='M150.131 114.16c.803-12.196 10.95-21.84 23.349-21.84 12.4 0 22.546 9.644 23.349 21.84h.051a3.12 3.12 0 0 1 3.12 3.12v3.12a3.12 3.12 0 0 1-4.362 2.863 23.464 23.464 0 0 1-14.361 14.527 7.8 7.8 0 0 1-15.594 0 23.464 23.464 0 0 1-14.36-14.527 3.12 3.12 0 0 1-4.362-2.863l-.001-3.12a3.12 3.12 0 0 1 3.171-3.12z'></path><g transform='translate(157.88 111.04)'><path fill='#E8CCC8' d='M9.36 15.6A4.68 4.68 0 1 1 0 15.6a4.68 4.68 0 0 1 9.36-.002M31.2 15.6a4.68 4.68 0 1 1-9.36.001 4.68 4.68 0 0 1 9.36-.002'></path><path fill='#D1BFA3' fillRule='nonzero' d='M16.885 8.633a.78.78 0 0 1-.32-1.527c1.048-.22 1.925-.05 2.408.759.938 1.575.611 3.39-.868 4.648-.86.731-2.066.993-2.909.483a.78.78 0 0 1 .808-1.334c.184.111.694 0 1.09-.337.906-.77 1.083-1.748.539-2.662-.051-.086-.277-.13-.748-.03z'></path><path fill='#635E59' fillRule='nonzero' d='M2.422 5.029a.78.78 0 1 1 1.396-.698C4.192 5.08 4.953 5.46 6.24 5.46c1.287 0 2.048-.38 2.422-1.129a.78.78 0 1 1 1.396.698C9.392 6.36 8.073 7.02 6.24 7.02s-3.152-.66-3.818-1.991zM21.142 5.029a.78.78 0 0 1 1.396-.698c.374.748 1.135 1.129 2.422 1.129 1.287 0 2.048-.38 2.422-1.129a.78.78 0 1 1 1.396.698C28.112 6.36 26.793 7.02 24.96 7.02s-3.152-.66-3.818-1.991z'></path><path fill='#FFF' d='M13.482 19.044c-.539.293-1.115.777-1.729 1.452-.92 1.013-.558 2.872.734 3.123 1.291.25 1.719-.858 1.83-1.432.075-.383-.204-1.43-.835-3.143z'></path><ellipse cx='15.6' cy='18.72' fill='#593A32' rx='4.68' ry='1.56'></ellipse></g><path d='M168.378 94.303a.78.78 0 0 1 1.504.415c-.157.567-.005.815 1.006 1.725 1.027.925 1.265 2.136.695 3.443a.78.78 0 1 1-1.43-.624c.306-.702.21-1.192-.309-1.66-1.422-1.28-1.836-1.959-1.466-3.3zM172.955 93.877a.78.78 0 1 1 1.256.927c-.35.473-.297.759.32 1.971.628 1.232.417 2.447-.584 3.463a.78.78 0 1 1-1.112-1.094c.538-.546.624-1.038.307-1.66-.869-1.705-1.012-2.488-.187-3.607zM177.44 93.84a.78.78 0 0 1 .907 1.27c-.478.341-.517.63-.304 1.973.216 1.365-.36 2.456-1.626 3.113a.78.78 0 1 1-.72-1.384c.681-.353.914-.795.805-1.485-.3-1.89-.194-2.678.937-3.487z' fill='#78706A' fillRule='nonzero'></path></g></g></g></svg>
   }
}

export default BannerIllustration
