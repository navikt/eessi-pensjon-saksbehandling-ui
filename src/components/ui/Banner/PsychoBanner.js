import React, { Component } from 'react'
import PT from 'prop-types'
import Psycho from '../Psycho/Psycho'
import * as Nav from '../Nav'

class PsychoBanner extends Component {
  render () {

    const { children } = this.props

    return <Nav.Veileder tekst={children}>
      <Psycho/>
    </Nav.Veileder>
  }
}

PsychoBanner.propTypes = {
    children: PT.node.isRequired
}

export default PsychoBanner
