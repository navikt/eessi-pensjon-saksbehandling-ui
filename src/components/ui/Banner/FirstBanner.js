import React, { Component } from 'react'
import { withNamespaces } from 'react-i18next'

import PsychoBanner from '../Banner/PsychoBanner'
import * as Nav from '../Nav'

import './Banner.css'

class FirstBanner extends Component {
  render () {
    const { t } = this.props

    return <div className='c-ui-banner'>
      <Nav.Sidetittel className='pt-5 pb-5 pl-5'>{t('app-headerTitle')}</Nav.Sidetittel>
      <PsychoBanner>
        <span>Sample text</span>
      </PsychoBanner>
    </div>
  }
}

export default withNamespaces()(FirstBanner)
