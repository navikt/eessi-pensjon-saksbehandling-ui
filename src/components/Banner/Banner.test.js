import React from 'react'
import Banner from './Banner'

describe('components/Banner', () => {

  let initialMockProps = {
    header: 'BANNER',
    t: jest.fn((translationString) => { return translationString }),
    toggleHighContrast: jest.fn()
  }

  it('Renders', () => {
    let wrapper = mount(<Banner {...initialMockProps} />)
    expect(wrapper.isEmptyRender()).toBeFalsy()
    expect(wrapper).toMatchSnapshot()
  })

  it('Has proper HTML structure', () => {
    let wrapper = mount(<Banner {...initialMockProps} />)
    expect(wrapper.exists('.c-banner')).toBeTruthy()
    expect(wrapper.find('.c-banner__title').hostNodes().render().text()).toEqual('BANNER')
    expect(wrapper.exists('#c-banner__highcontrast-link-id')).toBeTruthy()
  })

  it('Handles highContrast request', () => {
    let wrapper = mount(<Banner {...initialMockProps} />)
    wrapper.find('#c-banner__highcontrast-link-id').hostNodes().simulate('click')
    expect(initialMockProps.toggleHighContrast).toBeCalled()
  })
})
