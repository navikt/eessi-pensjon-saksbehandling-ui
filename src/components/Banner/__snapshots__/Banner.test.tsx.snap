// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Banner Renders 1`] = `
<Banner
  header={
    <h1>
      BANNER
    </h1>
  }
  labelHighContrast="mockLabel"
  onHighContrastClicked={[MockFunction]}
>
  <div
    className="c-banner"
    role="banner"
  >
    <Lenke
      className="c-banner__highcontrast-link mt-1"
      href="#highContrast"
      id="c-banner__highcontrast-link-id"
      onClick={[Function]}
    >
      <a
        className="lenke c-banner__highcontrast-link mt-1"
        href="#highContrast"
        id="c-banner__highcontrast-link-id"
        onClick={[Function]}
      >
        mockLabel
      </a>
    </Lenke>
    <div
      className="c-banner__title"
    >
      <h1>
        BANNER
      </h1>
    </div>
  </div>
</Banner>
`;
